ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts

```diff
@@ -51,8 +86,8 @@ for (const bondToolKey of Object.keys(BondTool)) {
       await page.close();
     });
 
-    test(`placing ${bondToolKey} on canvas`, async () => {
-      /*
+    test.only(`placing ${bondToolKey} on canvas`, async () => {
```

Comment:
are we sure we need this 'only'? 
As from documentation:
"When there are focused with "only" tests, only these tests run in entire project."

Reply:
Thank you for noticed. I removed `only`